notebooks/cleaning_data.ipynb

```diff
@@ -0,0 +1,2892 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Update indirect missing data percentages
</pre><p>same as before,&nbsp;<em>filtering/searching for labeled features</em>&nbsp;instead of&nbsp;<em>update indirect missing data.</em></p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mawada-sweis/Clustering-Analysis/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>